### PR TITLE
Add the news section to `README.md`

### PR DESCRIPTION
**Pull request by nabenabe0928**
## Motivation

This PR is to add the news section to `README.md` to increase the information connectivity.

## Description of the changes

- Add recent updates

### CODE REVIEW

**Review by HideakiImamura — APPROVED** [review]
LGTM.

**Comment by HideakiImamura** [comment]
@not522 Could you review this PR?

File: README.md [hunk 1]
```diff
@@ -20,6 +20,15 @@ for machine learning. It features an imperative, *define-by-run* style user API.
 *define-by-run* API, the code written with Optuna enjoys high modularity, and the user of
 Optuna can dynamically construct the search spaces for the hyperparameters.
 
+## :loudspeaker: News
+
+* **Aug 20, 2024**: We posted [an article](https://medium.com/optuna/file-management-during-llm-large-language-model-trainings-by-optuna-v4-0-0-artifact-store-5bdd5112f3c7) about [`ArtifactStore`](https://optuna.readthedocs.io/en/stable/reference/artifacts.html) officially supported from Optuna 4.0.
+* **Aug 15, 2024**: We posted [an article about `CatCMA`](https://medium.com/optuna/introduction-to-catcma-in-optunahub-ffa316309cb8), a sampler newly supported in [OptunaHub](https://hub.optuna.org/).
```
Comment (not522) [inline README.md:26]: How about changing the URL to a link to a CatCMA's page?

```suggestion
* **Aug 15, 2024**: We posted [an article about `CatCMA`](https://medium.com/optuna/introduction-to-catcma-in-optunahub-ffa316309cb8), a sampler newly supported in [OptunaHub](https://hub.optuna.org/samplers/catcma/).
```

**Review by not522 — APPROVED** [review]
LGTM!

**Pull request merged**